users/test_routes.py

```diff
@@ -47,10 +47,14 @@
 DUMMY_GAUTH_REQUEST_DATA = {"gauth_token": "fake_token_0123"}
 
 UPDATER_ID = "authorized-self-id"
-UPDATER_OBJECT = {
+AUTHORIZED_UPDATER = {
     "user_id": UPDATER_ID,
     "name": UPDATER_ID,
     "is_organizer": True}
+NON_AUTHORIZED_UPDATER = {
+    "user_id": UPDATER_ID,
```

Comment:
`UPDATER_ID` is `"authorized-self-id"` right now - maybe create a second ID as well and add both updaters to the DB - would simplify your tests as well because you don't have to update the updater user

Reply:
Done https://github.com/knative-portability/large-events/pull/41/commits/155b209516d739a785b9810fb7b136833c0d3d9e